helper/impls/helper-impl-110/src/main/java/com/linkedin/avroutil1/compatibility/avro110/FieldBuilder110.java

```diff
@@ -12,16 +12,22 @@
 
 
 public class FieldBuilder110 implements FieldBuilder {
-  private final Schema.Field _field;
+  private final String _name;
+  private Schema.Field _field;
   private Schema _schema;
   private String _doc;
   private Object _defaultVal;
   private Order _order;
 
   public FieldBuilder110(Schema.Field field) {
```

Comment:
i know this isnt your diff, but can we call this argument "other" ?

Reply:
Hi @radai-rosenblatt, are you referring to the **_field** to be changed to **_otherField**? If so I could also change it.

Reply:
no, the constructor argument